draft-ietf-rtcweb-jsep.xml

```diff
@@ -1818,8 +1870,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               "a=candidate" and "a=end-of-candidates" MUST be omitted.
               </t>
 
-              <t>For MediaStreamTracks that are still present, the "a=msid",
-              "a=ssrc", and "a=ssrc-group" lines MUST stay the same.</t>
+              <t>For RtpTransceivers with an RtpSender that is still
+              active, the "a=msid", "a=ssrc", and "a=ssrc-group" lines
```

Comment:
I don't think these values get to change just because the transceiver is no longer active (e.g. temporarily recvonly).